packages/invoices/src/OutboundShipment/ListView/Toolbar.tsx

```diff
@@ -13,7 +13,7 @@ import { useOutbound, OutboundRowFragment } from '../api';
 export const Toolbar: FC<{
   filter: FilterController;
 }> = ({ filter }) => {
-  const t = useTranslation('distribution');
+  const t = useTranslation('common');
```

Comment:
the `button.delete-lines` used in this component is in `distribution` and the `placeholder.search-by-name` which is in `common` is handled since common is a fallback